modules/blockChain/contractAddresses.ts

```diff
@@ -4,109 +4,131 @@ import type { ChainAddressMap } from './types'
+  [CHAINS.Holesky]: '',
 }
 
 export const AllowedRecipientReferralDaiRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xa295C212B44a48D07746d70d32Aa6Ca9b09Fb846',
   [CHAINS.Goerli]: '0x8fB566b1e78e603a86b97ada5FcA858764dF4088',
+  [CHAINS.Holesky]: '',
 }
 
 export const AllowedRecipientTrpLdoRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0x231Ac69A1A37649C6B06a71Ab32DdD92158C80b8',
   [CHAINS.Goerli]: '0x8C96a6522aEc036C4a384f8B7e05D93d6f3Dae39',
+  [CHAINS.Holesky]: '',
 }
 
 export const LegoLDORegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0x97615f72c3428A393d65A84A3ea6BBD9ad6C0D74',
   [CHAINS.Goerli]: '0x6342213719839c56fEe817539863aFB9821B03cb',
+  [CHAINS.Holesky]: '',
 }
 
 export const LegoDAIRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xb0FE4D300334461523D9d61AaD90D0494e1Abb43',
   [CHAINS.Goerli]: '0x5884f5849414D4317d175fEc144e2F87f699B082',
+  [CHAINS.Holesky]: '',
 }
 
 export const RccDAIRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xDc1A0C7849150f466F07d48b38eAA6cE99079f80',
   [CHAINS.Goerli]: '0x1440E8aDbE3a42a9EDB4b30059df8F6c35205a15',
+  [CHAINS.Holesky]: '',
 }
 
 export const PmlDAIRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xDFfCD3BF14796a62a804c1B16F877Cf7120379dB',
   [CHAINS.Goerli]: '0xAadfBd1ADE92d85c967f4aE096141F0F802F46Db',
+  [CHAINS.Holesky]: '',
 }
 
 export const AtcDAIRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xe07305F43B11F230EaA951002F6a55a16419B707',
   [CHAINS.Goerli]: '0xedD3B813275e1A88c2283FAfa5bf5396938ef59e',
+  [CHAINS.Holesky]: '',
 }
 
 export const gasFunderETHRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xCf46c4c7f936dF6aE12091ADB9897E3F2363f16F',
   [CHAINS.Goerli]: '0x0000000000000000000000000000000000000000',
+  [CHAINS.Holesky]: '0x0000000000000000000000000000000000000000',
 }
 
 export const StethRewardProgramRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0x48c4929630099b217136b64089E8543dB0E5163a',
   [CHAINS.Goerli]: '0x78797efCca6C537BF92FA6b25cBb14A6f1c128A0',
+  [CHAINS.Holesky]: '',
 }
 
 export const StethGasSupplyRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0x49d1363016aA899bba09ae972a1BF200dDf8C55F',
   [CHAINS.Goerli]: '0xF08a5f00824D4554a1FBebaE726609418dc819fb',
+  [CHAINS.Holesky]: '',
 }
 
 export const AragonACL: ChainAddressMap = {
   [CHAINS.Mainnet]: '0x9895F0F17cc1d1891b6f18ee0b483B6f221b37Bb',
   [CHAINS.Goerli]: '0xb3cf58412a00282934d3c3e73f49347567516e98',
+  [CHAINS.Holesky]: '0xfd1E42595CeC3E83239bf8dFc535250e7F48E0bC',
 }
 
 export const EVMScriptExecutor: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xF0211b7660680B49De1A7E9f25C65660F0a13Fea',
   [CHAINS.Goerli]: '0x3c9AcA237b838c59612d79198685e7f20C7fE783',
+  [CHAINS.Holesky]: '0x2819B65021E13CEEB9AC33E77DB32c7e64e7520D',
 }
 
 export const RewardsShareProgramRegistry: ChainAddressMap = {
   [CHAINS.Mainnet]: '0xdc7300622948a7AdaF339783F6991F9cdDD79776',
   [CHAINS.Goerli]: '0x8b59609f4bEa230E565Ae0C3C7b6913746Df1cF2',
+  [CHAINS.Holesky]: '',
```

Comment:
Why set all missing factories as empty strings? Isn't is the same as just not setting anything at all?

Reply:
I didn't test missing empty key.

Reply:
Would you mind to try testing that? Imo, it's better to not set empty strings here, since current implementation doesn't have checks for empty strings, but it has a check for an empty key (ref: https://github.com/lidofinance/easy-track-ui/blob/develop/modules/blockChain/utils/createContractHelpers.ts#L53)

Reply:
@katamarinaki doesn't work without keys

Reply:
@katamarinaki done